saleor/graphql/core/utils/error_codes.py

```diff
@@ -0,0 +1,182 @@
+from enum import Enum
+
+DJANGO_VALIDATORS_ERROR_CODES = [
```

Comment:
I would be glad if we were able to do it prettier. Unfortunately, Django doesn't store any list of errors or something. Every code that throws `ValidationError` uses string literals as the value of code parameter.
On the other hand, we can not drop those codes, because there are returned by model and forms validation.